cookbooks/Function-Calling-OpenAI/python/Function_Calling_with_OpenAI.ipynb

```diff
@@ -23,7 +23,7 @@
         "!pip install python-aiconfig\n",
         "\n",
         "# Create .env file at cookbooks/Function-Calling-OpenAI/python/.env containing the \n",
```

Comment:
I synced w. @rholinshead  and actually it turns out you CAN just do `OPENAI_API_KEY=<your key here>` without EXPORT. What I want to try is changing the `.env` file to be under `aiconfig/` dir, so could you change this line to this?

```suggestion
"# Create .env file at aiconfig/.env containing the following line:\n"
```